app/javascript/src/styles/specific/post_delete.scss

```diff
@@ -0,0 +1,62 @@
+#reason { width: 100%; }

```

Comment:
Can this ID be changed to something less generic, so that it doesn't cause surprises in the future if another element gets attached to an ID like this.